packages/core/config/babel-preset.js

```diff
@@ -21,6 +22,7 @@ module.exports = () => {
     presets: ['@babel/preset-react', '@babel/preset-typescript'],
     plugins: [
       ['@babel/plugin-proposal-class-properties', { loose: true }],
+      ['@babel/plugin-proposal-private-methods', { loose: true }],
```

Comment:
Could we add a note here: 
```suggestion
      // Note: The private method loose mode configuration setting must be the
      // same as @babel/plugin-proposal class-properties.
      // (https://babeljs.io/docs/en/babel-plugin-proposal-private-methods#loose)
      ['@babel/plugin-proposal-private-methods', { loose: true }],
```